app/wiring/AppLoader.scala

```diff
@@ -1,40 +1,49 @@
 package wiring
 
 import java.io.File
-
 import com.typesafe.scalalogging.StrictLogging
 import play.api.ApplicationLoader.Context
 import play.api._
 import play.api.libs.logback.LogbackLoggerConfigurator
 import com.gu.{AppIdentity, AwsIdentity, DevIdentity}
 import com.gu.conf.{ConfigurationLoader, FileConfigurationLocation, SSMConfigurationLocation}
+import services.Aws
+
+import scala.util.{Failure, Success, Try}
 
 class AppLoader extends ApplicationLoader with StrictLogging {
 
   override def load(context: Context): Application = {
     new LogbackLoggerConfigurator().configure(context.environment)
 
-    val identity = AppIdentity.whoAmI(defaultAppName = "admin-console")
-    val loadedConfig = ConfigurationLoader.load(identity) {
-      case AwsIdentity(app, stack, stage, _) => SSMConfigurationLocation(s"/$app/$stage")
-      case DevIdentity(app) =>
-        FileConfigurationLocation(new File(s"/etc/gu/support-admin-console.private.conf"))  //assume conf is available locally
+    def addConfigToContext(identity: AppIdentity): Try[Context] = Try {
+      val loadedConfig = ConfigurationLoader.load(identity) {
+        case AwsIdentity(app, stack, stage, _) => SSMConfigurationLocation(s"/$app/$stage", Aws.region.id())
+        case DevIdentity(app) =>
+          FileConfigurationLocation(new File(s"/etc/gu/support-admin-console.private.conf")) //assume conf is available locally
+      }
+
+      context.copy(initialConfiguration = context.initialConfiguration ++ Configuration(loadedConfig))
     }
 
-    val newContext = context.copy(initialConfiguration = context.initialConfiguration ++ Configuration(loadedConfig))
+    val application: Try[Application] = for {
+      identity <-
+        if (context.environment.mode == Mode.Dev) Success(DevIdentity("admin-console"))
+        else AppIdentity.whoAmI(defaultAppName = "admin-console", Aws.credentialsProvider.build())
+      newContext <- addConfigToContext(identity)
+      stage = identity match {
+        case AwsIdentity(_, _, s, _) => s
+        case _ => "DEV"
+      }
+      application <- Try(new AppComponents(newContext, stage).application)
+    } yield application
 
-    val stage = identity match {
-      case AwsIdentity(_, _, s, _) => s
-      case _ => "DEV"
-    }
 
-    try {
-      new AppComponents(newContext, stage).application
-    } catch {
-      case err: Throwable => {
+    application match {
```

Comment:
I've refactored this file, mainly because simple-configuration now returns a Try.
Also it's best practice not to call `whoAmI` in DEV mode because it blocks for a while trying to talk to aws